src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
Missing header comment. All non-trivial classes should have javadoc format header comments.